components/UserListItem.tsx

```diff
@@ -0,0 +1,101 @@
+import { View, Text, Image, TouchableOpacity } from "react-native";
+import React, { useState } from "react";
+import { SHADOWS } from "../theme";
+import { StackNavigationProp } from "@react-navigation/stack";
+import { useNavigation } from "@react-navigation/native";
+import { ListUser } from "./types/UserListTypes";
+import { handleSubscription } from "./HandleSubscription";
+
+type RootStackParamList = {
+  GeneralProfile: { username: string };
+  Error: { error: string };
+};
+
+type NavigationType = StackNavigationProp<RootStackParamList, "GeneralProfile">;
+
+type Props = {
+  user: ListUser;
+  subscriptionId?: string;
+};
+
+const UserListItem: React.FC<Props> = ({ user, subscriptionId }) => {
+  const navigation = useNavigation<NavigationType>();
+  const following = user.username;
+  const [isFollowed, setIsFollowed] = useState(subscriptionId !== "");
+  const subscriptionIdString =
+    subscriptionId !== undefined ? subscriptionId : "";
+  const [error, setError] = useState("");
+
+  // Nur relevant für Freundschaftsanfragen
+  // const handleAccept = () => {
+  //   console.log("Nutzer akzeptiert.");
+  // };
+
+  // const handleReject = () => {
+  //   console.log("Nutzer abgelehnt.");
+  // };
+
+  if (error !== "") {
+    navigation.navigate("Error", { error: error });
+  } else {
+    return (
+      <View
+        className="flex-row items-center rounded-3xl bg-white py-2 px-4 my-2 mx-6"
+        style={{ ...SHADOWS.small }}
```

Comment:
wieso setzt du den shadow mit style? Sollte mit tailwind doch auch gehen?

Reply:
siehe hier https://stackoverflow.com/questions/73372647/tailwind-css-in-react-native-box-shadow
style ist bei shadow die sichere variante + wir können ihn customizen